core/Azure.Mcp.Core/src/Services/Azure/Authentication/CustomChainedCredential.cs

```diff
@@ -40,7 +49,7 @@ public override ValueTask<AccessToken> GetTokenAsync(TokenRequestContext request
     private const string BrowserAuthenticationTimeoutEnvVarName = "AZURE_MCP_BROWSER_AUTH_TIMEOUT_SECONDS";
     private const string OnlyUseBrokerCredentialEnvVarName = "AZURE_MCP_ONLY_USE_BROKER_CREDENTIAL";
     private const string ClientIdEnvVarName = "AZURE_MCP_CLIENT_ID";
-    private const string IncludeProductionCredentialEnvVarName = "AZURE_MCP_INCLUDE_PRODUCTION_CREDENTIALS";
```

Comment:
Make sure this is listed in breaking changes and we should ship this before v1 IMO

Reply:
yep mentioned it.